apps/spruce/src/components/TextInputWithValidation/index.tsx

```diff
@@ -45,7 +47,7 @@ const TextInputWithValidation: React.FC<TextInputWithValidationProps> =
       ...rest
```

Comment:
[nit] I know this is an existing bug but can you reflect the `disabled` status in the `IconButton` too?